.circleci/config.yml

```diff
@@ -148,7 +148,10 @@ jobs:
       - write-version
       - run:
           name: Build Docker image
-          command: docker build -t app:build .
+          command: |
+            docker build \
+              -t app:build \
+              --build-arg VERSION="$(echo ${CIRCLE_SHA1} | cut -c -7)" .
```

Comment:
Note that I opted to use the short version of SHA1 here as it's identifying enough and can save a lot of space.